frame/support/procedural/src/pallet/expand/call.rs

```diff
@@ -177,6 +178,7 @@ pub fn expand_call(def: &mut Def) -> proc_macro2::TokenStream {
 			),
 			#(
 				#( #[doc = #fn_doc] )*
+				#[codec(index = #call_indices)]
```

Comment:
```suggestion
				#[codec(index = #call_indices)]
```

intentionally plural?